diff --git a/de.hama.lfs.repository/de.hama.lfs.platform.product b/de.hama.lfs.repository/de.hama.lfs.platform.product
index 3e43a0255..5083f963c 100644
--- a/de.hama.lfs.repository/de.hama.lfs.platform.product
+++ b/de.hama.lfs.repository/de.hama.lfs.platform.product
@@ -42,6 +42,11 @@
       <feature id="org.eclipse.equinox.p2.user.ui"/>
       <feature id="org.eclipse.equinox.p2.extras.feature"/>
       <feature id="de.hama.lfs.feature.platform"/>
+      <feature id="de.hama.lfs.feature.qs"/>
+      <feature id="de.hama.lfs.feature.packprogramm"/>
+      <feature id="de.hama.lfs.feature.konfiguration"/>
+      <feature id="de.hama.lfs.feature.auskunft"/>
+      <feature id="de.hama.lfs.feature.auftragsbearbeitung"/>
       <feature id="org.eclipse.platform"/>
       <feature id="org.eclipse.equinox.p2.core.feature"/>
    </features>
